gradle.properties

```
@@ -1,2 +1,5 @@
 loom.platform=forge
 org.gradle.jvmargs=-Xmx2g
+baseGroup = com.example
+mcVersion = "1.8.9"
+version = "1.0.0"
```

I don't think these properties should be quoted: 

```json
[
  {
    "modid": "examplemod",
    "name": "Xample Mod",
    "description": "A mod that is used as an example.",
    "version": ""1.0.0"",
``` 

```suggestion
mcVersion = 1.8.9
version = 1.0.0
```

---

Aside from this, this pr looks ready.